README.md

```diff
@@ -121,7 +121,7 @@ pip install -e .
 
 ## 🧬 How to use
 
-```python
+```
```

Comment:
Can you please re-render this and make sure that the python suffix stays there? Otherwise the syntax highlighting gets lost.

Reply:
So, the nbdev rendering does not keep the python language specifier (L/? Do I have to manually ensure that the python L/S is still there (after the rendering)?

Reply:
Did you save the executed notebook and ran the build_docs? The only reason I can think of that it would do this is if there's no metadata specifying the language in the notebook.

Reply:
Ah! No, I did not execute the notebook. I just made the changes in the markdown and saved the file. And then ran the command you had mentioned.

Reply:
Following #28 I think we may change the way we generate the README so if you want to fix the links as well we appreciate it but if this becomes too burdensome the changes in the manifest are already a huge help. Thank you!

Reply:
I had a similar problem for one of my repositories. The hack that I implemented allowed me to keep relative links in readme file, but while publishing to PyPI, since it runs setup.py, the target set of links were updated with absolute path. This might give you an idea.

- https://github.com/sugatoray/genespeak/blob/e6b49491cf5a872a03d55403ed34d2fa3469d0b8/setup.py#L7-L37

I am skipping updating the notebook/readme for now (reverted already). Please let me know if the rest is fine now.